src/apps/api/views/competitions.py

```diff
@@ -34,15 +36,24 @@ def get_queryset(self):
             qs = qs.filter(participants__user=self.request.user, participants__status="approved")
 
         # On GETs lets optimize the query to reduce DB calls
-        if self.request.method == 'GET' and self.action != 'list':
-            qs = qs.select_related('created_by')
-            qs = qs.prefetch_related(
-                'phases',
-                'pages',
-                'leaderboards',
-                'leaderboards__columns',
-                'collaborators',
-            )
+        if self.request.method == 'GET':
+            if self.action == 'list':
+                qs = qs.select_related('created_by')
```

Comment:
can do `pass` here and factor the `qs = qs.select_related('created_by')` out of both conditionals in this if.

could also not do `pass` and `if self.action != 'list':` I believe?